### PR TITLE
Add GraphQL tests for Netflow operations

### LINKED ISSUE #1429: Add GraphQL tests for Netflow operations
Add tests for representative Netflow queries/mutations. Follow parent issue requirements.

### PR DESCRIPTION
**Pull request by dayeon5470**
Close #1429

### CODE REVIEW

File: src/graphql/netflow.rs [hunk 1]
```diff
@@ -477,4 +765,94 @@ mod tests {
...
+                None,
+                None,
+                None,
+            )
+            .unwrap();
+        assert!(!reject_contents);
+    }
+
+    #[test]
+    fn test_tcp_flags_returns_none_when_empty() {
```
Comment (henry0715-dev) [inline src/graphql/netflow.rs]: It would be good to strengthen the unit tests for the `tcp_flags` function.
Adding more test cases would help ensure correctness across different flag combinations and edge cases.

Comment (dayeon5470) [reply src/graphql/netflow.rs]: I’ve added it.

**Review by henry0715-dev — APPROVED** [review]

**Comment by henry0715-dev** [comment]
Test coverage for local (non-cluster) data responses is limited. The existing tests primarily validate timestamps and do not assert field-level transformations (e.g., `tcp_flags`, `first/last` conversions, `tos` hex conversion, and `sampling_mode` string mapping).

**Comment by henry0715-dev** [comment]
The test names `netflow5_timestamp_fomat_stability` and `netflow9_timestamp_fomat_stability` contain typos. Please rename them to `netflow5_timestamp_format_stability` and `netflow9_timestamp_format_stability`.

**Comment by dayeon5470** [comment]
I’ve updated everything.
